Extensions/PlayerAuthentication/playerauthenticationtools.ts

```diff
@@ -36,6 +37,36 @@ namespace gdjs {
       _justLoggedIn = false;
     });
 
+    // If the extension is used, register an eventlistener to know if the user is
+    // logged in while playing the game on Liluo.io.
+    // Then send a message to the parent iframe to say that the player auth is ready.
+    gdjs.registerFirstRuntimeSceneLoadedCallback(
+      (runtimeScene: RuntimeScene) => {
+        _authenticationMessageCallback = (event: MessageEvent) => {
```

Comment:
Is there a risk that `_authenticationMessageCallback` is not null, and so erased and the old callback then leaking forever as already attached to a window.addEventListener?
Would calling `removeAuthenticationCallbacks` be safer?

Reply:
We could call it for safety, but I don't see how this could happen as this should be the first thing to be called.
I'll do it anyway